featurebyte/api/feature.py

```diff
@@ -695,6 +697,8 @@ def save(self, conflict_resolution: ConflictResolution = "raise") -> None:
         conflict_resolution: ConflictResolution
             "raise" will raise an error when we encounter a conflict error.
             "retrieve" will handle the conflict error by retrieving the object with the same name.
+        _id: Optional[ObjectId]
+            The object ID to be used when saving the object. If not provided, a new object ID will be generated.
```

Comment:
This introduction of `_id` parameter here is necessary. Otherwise, there is no ways to specify `id` in feature definition file. Without specifying the `_id`, it will cause changes in feature `_id` before & after batch feature creation.